src/Components/ETRA/EmailSignOff/index.tsx

```diff
@@ -0,0 +1,167 @@
+import { IIssue } from '../../../Domain/Issues'
+import { IServiceProvider, ServiceContext } from '../../../ServiceContext';
+import { ISaveMeetingDraftUseCase } from '../../../Boundary/SaveMeetingDraft';
+import { ISignOffMeetingUseCase } from '../../../Boundary/SignOffMeeting';
+import { ICreateMeetingUseCase } from '../../../Boundary/CreateMeeting'
+import { MeetingModel, IMeetingModel } from '../../../Domain/Meeting';
+import { IAttendees } from '../../../Domain/Attendees';
+import { ISignOff } from '../../../Domain/SignOff';
+import { Redirect, Link } from 'react-router-dom';
+import Attendees from '../../Attendees';
+import RecordIssues from '../../RecordIssues';
+
+export interface ISaveMeetingProps {
+  signOffMode: boolean,
+  traId: number,
+  meetingId?: string,
+  meetingName: string,
+  attendees: IAttendees,
+  issues: Array<IIssue>,
+  signOff: ISignOff,
+  isSessionLive?:boolean
+}
+
+export interface ISaveMeetingState {
+  isAttemptingToSave: boolean;
+  isValid: boolean;
+  redirectToLandingPage: boolean;
+ 
+}
+
+export interface IUnreviewedMeetingModel {
+  traId: number;
+  meetingName: string;
+  issues: Array<IIssue>;
+  meetingAttendance: IAttendees;
+}
+
+export class UnreviewedMeetingModel implements IUnreviewedMeetingModel {  
+  public readonly traId: number;
+  meetingName: string;
+  issues: IIssue[];
+  meetingAttendance: IAttendees;
+
+  constructor(traId: number, meetingName: string, issues: IIssue[], attendees: IAttendees){
+      this.traId = traId;
+      this.meetingName = meetingName;
+      this.issues = issues;
+      this.meetingAttendance = attendees;
+  }
+}
+
+
+export class EmailSignOff extends React.Component<ISaveMeetingProps, ISaveMeetingState> {
+  public static contextType = ServiceContext;
+  private readonly createMeeting: ICreateMeetingUseCase;
+  public constructor(props: ISaveMeetingProps, context: IServiceProvider) {
+    super(props, context);
+    this.createMeeting = context.get<ICreateMeetingUseCase>("ICreateMeetingUseCase");
+    
+    this.state = {
+      isAttemptingToSave: false,
+      isValid: this.checkIsValid(this.props),
+      redirectToLandingPage: false
+    }
+  }
+ 
+  componentWillReceiveProps(newProps: ISaveMeetingProps){
+    this.setState({isValid: this.checkIsValid(newProps)})
+   
+  }
+
+  private checkIsValid(props: ISaveMeetingProps){
+    if(isNaN(props.attendees.attendees) || props.attendees.attendees < 1){
+      return false;
+    }
+    
+    return true;
+  }
+
+  getUnreviewedMeetingModel = () : IUnreviewedMeetingModel => {
+    return new UnreviewedMeetingModel(
+      this.props.traId,
+      this.props.meetingName,
+      this.props.issues,
+      this.props.attendees
+    );
+  }
+
+  handleReviewLater = async () => { 
+    this.setState({ isAttemptingToSave: true });
+    /*const successful = await this.createMeeting.Execute(this.getUnreviewedMeetingModel());
+
+    if (successful) {
+      this.props.onReviewLater();
+    }
+    else {
```

Comment:
maybe remove all this commented code if not needed anymore?

Reply:
I will do. I'm just getting an understanding of how the code works. 
It's easier for me to uncomment stuff if I think I need it.